File: src/main/java/com/sfh/pokeRogueBot/model/modifier/impl/PokemonExpBoosterModifierItem.java
package com.sfh.pokeRogueBot.model.modifier.impl;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class PokemonExpBoosterModifierItem extends PokemonHeldItemModifierItem {

    public static final String TARGET = "PokemonExpBoosterModifierType";

    private int boostPercent;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/modifier/impl/PokemonExpBoosterModifierItem.kt
package com.sfh.pokeRogueBot.model.modifier.impl

class PokemonExpBoosterModifierItem : PokemonHeldItemModifierItem() {

    companion object {
        const val TARGET = "PokemonExpBoosterModifierType"
    }

    var boostPercent: Int = 0
}